### PR TITLE
Use cross-fetch instead of node-fetch & more typescript declarations

### PR DESCRIPTION
**Pull request by dvlkv**
Changelist:
 - replaced node-fetch with [cross-fetch](https://github.com/lquixada/cross-fetch)
 - fixed typo in application/* files
 - Added profile type declarations for Client.js

### CODE REVIEW

File: src/structures/agent.d.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
-import fetch from "node-fetch"
+import fetch  from "cross-fetch"
+import { Profile } from "../profiles";
```
Comment (seia-soto) [inline src/structures/agent.d.ts]: Please, remove semicolon at this line.

**Comment by seia-soto** [comment]
Sorry, I am bit late and just woke up. I just searched what's different between node-fetch, isomorphic-fetch, and cross-fetch. I think your choice is really nice as last time. Thank you for PR.

File: src/structures/agent.d.ts (hunk 1 above)
Comment (dvlkv) [reply src/structures/agent.d.ts]: Sorry :)

**Comment by seia-soto** [comment]
Really thanks!

**Pull request merged**